prac_05/hex_colours.py

```diff
@@ -0,0 +1,30 @@
+# Adrianto Hartono
+# 14347930
+
+"""
+Hex Colours
+"""
+
+COLORS = {
+    "ACIDGREEN": "	##b0bf1a",
+    "ALICEBLUE": "#f0f8ff",
+    "AQUA": "#00ffff",
+    "APRICOT": "#fbceb1",
+    "BABYBLUE": "#89cff0",
+    "BEIGE": "#f5f5dc",
+    "BLACK": "#000000",
+    "CAMEL": "#c19a6b",
+    "CRYSTAL": "#a7d8de",
+    "FAWN": "#e5aa70"
+}
+
+color_name = input("Enter the color name: ").upper()
+
+while color_name != "":
+    if color_name in COLORS:
+        print(f"The hexadecimal color code for {color_name} is {COLORS[color_name]}")
```

Comment:
Detailed and clear print statement to identify colour code, well done!

Reply:
thank you for the feedback :)

Reply:
Nice code just remove the extra space on line 9 and your code is perfect

Reply:
okay noted, thanks again for the feedback 🙏